src/MDMTransitionContext.h

```diff
@@ -80,3 +80,9 @@ NS_SWIFT_NAME(TransitionContext)
 @property(nonatomic, strong, readonly, nonnull) UIView *containerView;
 
 @end
+
+NS_SWIFT_NAME(InteractiveTransitionContext)
+@protocol MDMInteractiveTransitionContext
```

Comment:
This flag is available in the `MDMTransitionContext` protocol already, so if we were able to make this interaction context conform to that protocol we could take advantage of those existing APIs.